### ISSUE TITLE
Node Graph

### ISSUE THREAD

**Issue by MagikEh** [MetPX/sarracenia#562]
Currently, the only visualization aid to the data flow of a sarracenia node is that made by whoever set it up.

sr3 deals with all the configuration files (on a per-node basis) at once in memory, so it'd be a case of dumping that in-memory structure to some form of a graph that would enable this vision.

@reidsunderland has already done some work with a custom python script that scrapes configs and dumps to [mermaid graphs](https://mermaid-js.github.io/mermaid/#/), however, that work has also shown the limits of gitlab's and github's ability to parse more than 5,000 lines.

**Comment by petersilva** [comment]
it would be great to take the result of *sr3 dump* and use that to build graphs.  especially sr3 dump from multiple machines.  One thing that would make that work easier is to complete #634

**Comment by petersilva** [comment]
note: sr dump also exists... but it is even further from JSON.

**Comment by reidsunderland** [comment]
yes, we definitely need to use the sr3 config parser to ensure that configs and include files are parsed correctly.

My rough script from last year only looked at the exchange and post_exchange in .conf files, ignoring brokers, include files and and configs that don't explicitly define which exchange is used.

**Comment by petersilva** [comment]
Good news! @Rehouma63  is working on #634
